runtime/centrifuge/src/lib.rs

```diff
@@ -258,7 +262,14 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 				| pallet_xcm::Call::force_subscribe_version_notify { .. }
 				| pallet_xcm::Call::force_unsubscribe_version_notify { .. } => true,
 			},
-			RuntimeCall::EVM(_) => false, // Disable all non-root EVM access
+			// Disable all non-root EVM access
+			RuntimeCall::EVM(_) => false,
+			// We block this call since it includes Moonbeam trait implementations such
+			// as UtilityEncodeCall and XcmTransact that we don't implement and don't want
+			// arbitrary users calling it.
+			RuntimeCall::XcmTransactor(
```

Comment:
Copied over from the development runtime, I assume this point still makes sense here, please let me know otherwise.

Reply:
As long as our EVM codes do not call this it is fine.

Reply:
Got it, thank you!